src/com/tigersale/userInterface/customerUser/CustomerUserAddressView.java

```diff
@@ -0,0 +1,135 @@
+            System.out.println("0: Go Back");
+            System.out.println("1: Add Address Information");
+            System.out.println("2: Update Address Information");
+            System.out.println("3: View Current Address Information");
+
+            // Try to get a numeric response from the user
+            try {
+
+                choice = scanner.nextInt();
+            } catch (InputMismatchException e) {
+                System.out.println("Enter an integer corresponding to your preferred option.");
+                scanner.next();
+                continue;
+            }
+
+            switch (choice) {
+
+
+                // Exit this view
+                case 0:
+                    return;
+
+                case 1:
+                    System.out.println("Please enter your street address.");
+                    System.out.flush();
+                    String street = scanner.next();
+
+                    System.out.println("Please enter your city.");
+                    System.out.flush();
+                    String city = scanner.next();
+
+                    System.out.println("Please enter your state.");
+                    System.out.flush();
+                    String state = scanner.next();
+
+                    System.out.println("Please enter your zip code.");
+                    System.out.flush();
+                    String zip = scanner.next();
+                    //CustomerUser user = CustomerUserTable.login(username, password);
+
+                    System.out.println("You have successfully added an address.\n");
+                    insertAddress(street, city, state, zip, user);
+
+                    break;
+
+                // Update an existing address
+                case 2:
+                    System.out.println("Your current addresses:");
+                    int addrNum = 0;
+                    List<Address> addressList = getAddresses(user);
+
+                    for(Address adr: addressList){
+                        System.out.println("\nAddress #" + addrNum);
+                        System.out.println("Street:\t\t" + adr.street);
+                        System.out.println("City:\t\t" + adr.city);
+                        System.out.println("State:\t\t" + adr.state);
+                        System.out.println("Zip Code:\t" + adr.zipCode);
+                        addrNum++;
+                    }
+
+                    System.out.println("Enter the address # to remove.");
+
+                    int killChoice = 0;
+                    // Try to get a numeric response from the user
+                    try {
+
+                        killChoice = scanner.nextInt();
+                    } catch (InputMismatchException e) {
+                        System.out.println("Error! Input must be an integer.\n");
+                        break;
+                    }
+
+                    if(killChoice >= addressList.size()){
```

Comment:
You need to give the option to "Go Back"   so "0 - To go back" and then the addresses